automan/api/projects/storages/views.py

```diff
@@ -57,7 +57,18 @@ def upload(self, request, project_id):
         key = request.data.get('key')
         storage_manager = StorageManager(project_id, storage_id)
         bucket = storage_manager.storage['storage_config']['bucket']
-        res = storage_manager.get_s3_presigned_url(bucket, key)
+        if storage_manager.original_file_exists(key):
+            res = {
+                'can_upload': False,
+                'message': 'the file exists'
+            }
+            return HttpResponse(status=200,
```

Comment:
Should send status code as `409`. and can_upload is not needed.
https://developer.mozilla.org/ja/docs/Web/HTTP/Status/409